backend/ASTU-backend-group-1/delivery/routers/BlogRouter.go

```diff
@@ -0,0 +1,70 @@
+package router
+
+import (
+	"astu-backend-g1/delivery/controllers"
+
+	"github.com/gin-gonic/gin"
+)
+
+type BlogRoute struct {
+	usecase controllers.BlogController
+	authController controllers.GeneralAuthorizationController
+}
+
+func NewBlogRoute(usecase controllers.BlogController,authcontroller controllers.GeneralAuthorizationController) *BlogRoute {
+	return &BlogRoute{usecase: usecase,authController: authcontroller}
+}
+
+func (gr *BlogRoute) GinBlogRouter() {
+	router := gin.Default()
+	blogRouter := router.Group("/blog")
```

Comment:
Instead of using ` /blog ` it should be `/blogs` and the all method is not necessary